src/routing_filter.c

```diff
@@ -8,7 +8,7 @@
  *     This file contains the implementation for a routing filter
  *----------------------------------------------------------------------
  */
-
+#include <unistd.h>
```

Comment:
Platformize the functionality you need here.

Reply:
This #include seems like left-over cruft. 

I went back and took a look at the diffs introduced, which is just an elaborated print statement.

I don't need L11 anymore, so took it out.  Dev builds seem to work ok, will cross-check in CI.

Thanks for catching this ...

Reply:
@rtjohnso --  I had to re-introduce this `#include`. It's needed to resolve extern reference to `getpid()` that is used in the error message. Otherwise, CI quick_check job failed in clang debug build mode. 

[ Strangely, I can't repro that failure on my Linux dev box in the exact same build mode. ]

This isn't new with this change-set. Other files are also #include'ing `unistd.h` whenever `getpid()` is used.

Reply:
I think all the other instances of `#include <unistd.h>` were introduced by you in the previous PR for multi-process support.

We need to clean them all out by platformizing `getpid`.

I'd prefer to do it now, but I understand that doing it now may create additional difficulty in rebasing and merging the third PR in this line of work.

Would you mind doing it as a follow-on PR after we land the third one?

Reply:
@rtjohnso - I peeled this cleanup item out on its own new issue. See issue #599 where I've described a possible solution.

I will clean this up after part-3 lands.